src/coreclr/tools/ILVerification/ILImporter.Verify.cs

```diff
@@ -2647,6 +2648,18 @@ void CheckPendingPrefix(Prefix mask)
             Check((mask & Prefix.Constrained) == 0, VerifierError.Constrained);
         }
 
+        static bool HasIsExternalInit(MethodSignature signature)
+        {
+            if (signature.HasEmbeddedSignatureData && signature.GetEmbeddedSignatureData() != null)
```

Comment:
```suggestion
            if (signature.HasEmbeddedSignatureData)
```
Redundant with HasEmbeddedSignatureData

Reply:
Deleted from R2R test too.